### ISSUE TITLE
Script functionality broken in 3.2.3[x]

### ISSUE THREAD

**Issue by Sharlikran** [TES5Edit/TES5Edit#478]
Without any ESL files loaded the program may still assume that IwbFile.LoadOrder has anything to do with the upper bytes of FormIDs.

```Pascal
{
  Copy XPRI subrecord (attached papyrus scripts data) to selected record(s)
  from provided source record.
}
unit UserScript;

function Process(e: IInterface): Integer;
var
  m: IInterface;
begin
  if Signature(e) <> 'CELL' then
    Exit;
  
  m := MasterOrSelf(e);
  
  if Equals(e, m) then
    Exit;
  
  if not ElementExists(m, 'XPRI') then
    Exit;
  
  Add(e, 'XPRI', True);
  ElementAssign(ElementByPath(e, 'XPRI'), LowInteger, ElementByPath(m, 'XPRI'), False);
end;

end.
```

For example that will work with SSEEdit-6557c88 from commit f39199fbf719b4d4ef18ec16dba303d563132774 but not 3.2.3h.

**Comment by eFrysTon** [comment]
Describe "broken"? Nothing in that script should fail, even if ESLs are involved...

**Comment by Sharlikran** [comment]
What I expected was that as the script cycled through all the CELL records it would forcefully copy XPRI from the Master to the plugin. The CK breaks that record.  

What happens instead is that the script doesn't copy XPRI from the master as expected.

Expected:
![image](https://user-images.githubusercontent.com/1912426/43043516-49104842-8d4a-11e8-9d86-aeaa01ea5e40.png)

Current Result (XPRI not copied from master):
![image](https://user-images.githubusercontent.com/1912426/43043522-5ecd3a28-8d4a-11e8-99f4-8cc0e07825ff.png)

**Comment by eFrysTon** [comment]
The script seems to work fine to me?

Make sure that you have selected the record in the module where you want it to be overwritten, not in the module that contains the master record.

In your screenshot above you have selected the record inside DLCNukaWorld, in that case your if Equals(e, m) then Exit; check will exit, because e and the "MasterOrSelf" are the same record. If you select the record of the same FormID inside UltraInteriorLightning.esp instead, then the script will copy correctly.

**Comment by zilav** [comment]
Everything works fine for sure.

**Comment by Sharlikran** [comment]
I'll make sure I didn't mess up and select the master as you mentioned.

**Comment by Sharlikran** [comment]
Yep it looks like I might be selecting the DLC and not the plugin like Elminster said.
